tensornetwork/backends/tensorflow/tensorflow_backend.py

```diff
@@ -383,3 +385,15 @@ def sign(self, tensor: Tensor) -> Tensor:
       tensor: The input tensor.
     """
     return tf.math.sign(tensor)
+
+  def item(self, tensor):
+    numel = 0
+    if len(tensor.shape) > 0:
+      numel = reduce(mul, tensor.shape)
```

Comment:
We generally prefer using `np.prod`.

Reply:
np.prod has a significantly larger overhead for small sequences, this is why I'm using reduce here

Reply:
it's not important though

Reply:
If it's actually faster than we can keep it.